src/pybind/mgr/dashboard/.pylintrc

```diff
@@ -117,7 +117,8 @@ disable=import-star-module-level,
         too-few-public-methods,
         no-member,
         too-many-arguments,
-        too-many-locals
+        too-many-locals,
+        too-many-statements
```

Comment:
do we really need to disalble `too-many-statements` application-wide??

Reply:
Please discuss this here #22528 (this PR is imported only).